test/python/test_approximate_equivalence.py

```
@@ -0,0 +1,156 @@
+        phase,
+        parallel=False,
+        check_approximate_equivalence=True,
+        approximate_checking_threshold=0.49,
+    )
+
+    assert result.equivalence == EquivalenceCriterion.not_equivalent
+    assert result.started_simulations == 0
+
+
+@pytest.mark.parametrize("parallel", [False, True])
+def test_default_simulation_is_not_used_for_process_distance(parallel: bool) -> None:
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)